modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -49,6 +49,13 @@ menuconfig BUILD_WITH_TFM
 
 if BUILD_WITH_TFM
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
+	default y
```

Comment:
I think we should follow TF-M default here, which seems to be off per default.
See comment: https://github.com/zephyrproject-rtos/zephyr/pull/38670#discussion_r712969465
```suggestion
```

Then the nrf9160 and nrf5340 board Kconfigs can default to `y` if they like, for example inside this safe guard:
https://github.com/zephyrproject-rtos/zephyr/blob/56018056955c1713ebac24724b8b0126a05dc5c9/boards/arm/nrf9160dk_nrf9160/Kconfig.defconfig#L17
https://github.com/zephyrproject-rtos/zephyr/blob/56018056955c1713ebac24724b8b0126a05dc5c9/boards/arm/nrf5340dk_nrf5340/Kconfig.defconfig#L17

Reply:
+1